core/base/src/TSystem.cxx

```diff
@@ -1839,6 +1839,46 @@ static bool R__MatchFilename(const char *left, const char *right)
 #endif
 }
 
+
+////////////////////////////////////////////////////////////////////////////////
+/// Check if lib is in the dynamic linker cache, returns true if it is.
```

Comment:
```suggestion
/// Check if lib is in the dynamic linker cache, returns true if it is, and if so,
/// modifies the library file name parameter `lib` from `/usr/lib/libFOO.dylib`
/// to `-lFOO` such that it can be passed to the linker.
```